src/newgrf.cpp

```diff
@@ -400,7 +98,7 @@ void GrfMsgI(int severity, const std::string &msg)
  * @param grfid The grfID to obtain the file for
  * @return The file.
  */
-static GRFFile *GetFileByGRFID(uint32_t grfid)
+GRFFile *GetFileByGRFID(uint32_t grfid)
```

Comment:
Most of the formerly `static` methods from this file probably only work during GRF loading.
E.g. `GetFileByGRFID` and should not be confused with `FindGRFConfig`.
Can we put these functions into some separate namespace?

Reply:
Had a couple of goes at namespacing things but it's a bit tricky as it's all spread around. Not sure the best approach tbh.

Reply:
I guess the prototypes being in `newgrf_internal.h` is good enough.